class-wc-calypso-bridge.php

```diff
@@ -91,9 +92,31 @@ public static function instance() {
 	public function enqueue_calypsoify_styles() {
 		$asset_path = self::$plugin_asset_path ? self::$plugin_asset_path : self::MU_PLUGIN_ASSET_PATH;
 		wp_enqueue_style( 'wc-calypso-bridge-calypsoify', $asset_path . 'assets/css/calypsoify.css', array(), WC_CALYPSO_BRIDGE_CURRENT_VERSION, 'all' );
+		wp_enqueue_script( 'wc-calypso-bridge-calypsoify', $asset_path . 'assets/js/calypsoify.js', array( 'jquery' ), WC_CALYPSO_BRIDGE_CURRENT_VERSION, true );
 		add_filter( 'woocommerce_display_admin_footer_text', '__return_false' );
 	}
 
+	/**
+	 * Track Calypsoify events when turned on or off
+	 */
+	public function track_calypsoify_toggle() {
+		if ( isset( $_GET['calypsoify'] ) ) { // WPCS: CSRF ok.
+			$current_user      = wp_get_current_user();
+			$calypsoify_status = (int) get_user_meta( $current_user->ID, 'calypsoify', true );
+			if ( 1 === $calypsoify_status && 0 === (int) $_GET['calypsoify'] ) { // WPCS: CSRF ok.
+				jetpack_tracks_record_event(
+					$current_user,
+					'atomic_wc_calypsoify_off'
```

Comment:
While not explicitly addressed in the "Tracks Event Design & Naming Conventions" field guide page, there is a section in the FG that talks about not using variables in the name such as post type, and instead use a single event with a prop. I feel like that can apply here. 

So I'm wondering if maybe we should name this `atomic_wc_calypsoify_toggle` instead with a prop for off/on.

Reply:
That seems like a good idea plus we can clean up the logic here a bit.  https://github.com/Automattic/wc-calypso-bridge/pull/133/commits/001ffba4bc7f194019218b4ee1db1322d09ab74b